### DISCUSSION TITLE
Some issue with the commands

### DISCUSSION THREAD

**Discussion by Sahil1776** [open-edge-platform/anomalib#1417] [Q&A]
I have read the Anomalib documentation and everything seems to make sense for me. 
I trained the EfficientAd algorithm on a small dataset using Anomalib just to see how it works. 

The command mentioned in the documentation is not working. I was following the notebooks provided but I am missing something with the commands in documentation. Here's an example:

I trained my model. I have the checkpoint file, xml, json etc files. I tried to obtain inference using this command and this is appearing:

<img width="1440" alt="Screenshot 2023-10-15 at 9 22 19 PM" src="https://github.com/openvinotoolkit/anomalib/assets/48206391/d27d8c9c-8f77-456f-8b28-b8d10ae13ef8">

I highly appreciate if someone could help me with this.

**Comment by kikyostudypython** [comment]
Wrong command.     openvino_inferencer.py, not openvino.py
